profile.js

```diff
@@ -35,4 +35,64 @@ document.getElementById('menu').addEventListener('click', () => {
   document.getElementById('link1').addEventListener('click', hideMobMenu);
   document.getElementById('link2').addEventListener('click', hideMobMenu);
   document.getElementById('link3').addEventListener('click', hideMobMenu);
-});
\ No newline at end of file
+});
+// || =========== Pop-Up Modal ============ ||
+const arrProjects = [];
+function addProjects() {
+  // Stores Project Infos in arrProject
+  const colTitles = document.getElementsByClassName('project-titles');
+  const colClient = document.getElementsByClassName('client');
+  const colRole = document.getElementsByClassName('type');
+  const colYear = document.getElementsByClassName('year');
+  for (let i = 0; i < colTitles.length; i += 1) {
+    // | creating a new project object an pushing it to arrProject
+    const objProject = {};
+    objProject.name = colTitles[i].textContent;
+    objProject.client = colClient[i].textContent;
+    objProject.role = colRole[i].textContent;
+    objProject.year = colYear[i].textContent;
+    objProject.tags = ['html', 'css', 'javascript', 'github', 'ruby', 'bootstrap'];
+    objProject.snapshot = `images/SnapshootD${i + 1}.jpg`;
+    arrProjects.push(objProject);
+  }
+}
+
+function getProjectDetails(proID) {
+  // Retrieving  Project Info
+  return arrProjects[proID];
+}
+
+function showProjectDetails(proID) {
+  // showing  Project Info
+  const modPopUp = document.getElementById('modal-ui-wrapper');
+  const p = getProjectDetails(proID - 1);
+  document.getElementById('mc-title').textContent = p.name;
+  document.getElementById('mc-client').textContent = p.client;
```

Comment:
- [ ] Please use the correct font-weight for the project description and make sure it matches the Template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/169248974-59a33c15-a87c-473c-8af3-f58f10921030.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/169248934-338784a8-3ae1-4da2-afd8-60eb102dbd2e.png)